### ISSUE TITLE
FAIL: Ensure the font supports case swapping for all its glyphs.

### ISSUE THREAD

**Issue by vv-monsalve** [petrvanblokland/TYPETR-Bitcount#15]
<details><summary>🔥 <b>FAIL:</b> Ensure the font supports case swapping for all its glyphs. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/case_mapping">com.google.fonts/check/case_mapping</a>)</summary><div>

* 🔥 **FAIL** The following glyphs lack their case-swapping counterparts:

| Glyph present in the font | Missing case-swapping counterpart |
| :--- | :--- |
| U+0162: LATIN CAPITAL LETTER T WITH CEDILLA | U+0163: LATIN SMALL LETTER T WITH CEDILLA |

 [code: missing-case-counterparts]
</div></details>

This Fail is reported for all the [new fonts in the CI run](https://github.com/petrvanblokland/TYPETR-Bitcount/actions/runs/8650173142/artifacts/1405875842).

**Comment by vv-monsalve** [comment]
This is still reported for all the fonts from the CI build at commit `806c71d`.

**Comment by vv-monsalve** [comment]
Our glyphsets no longer require the `T + cedilla` codepoints. The solution here would be to delete the `0x0162` one.

**Comment by vv-monsalve** [comment]
This previous case for The `0163: LATIN SMALL LETTER T WITH CEDILLA` was fixed.

However, the Fail is still reported for the following fonts at the latest CI Run at commit `560a399` for a new letter:


<table>
<thead>
<tr>
<th align="left">Glyph present in the font</th>
<th align="left">Missing case-swapping counterpart</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left">U+0145: LATIN CAPITAL LETTER N WITH CEDILLA</td>
<td align="left">U+0146: LATIN SMALL LETTER N WITH CEDILLA</td>
</tr>
</tbody>
</table>

Affected fonts:

- BitcountGridSingleInk[CRSV,ELSH,ELXP,SZP1,SZP2,XPN1,XPN2,YPN1,YPN2,slnt,wght].ttf
- BitcountSingle[CRSV,ELSH,ELXP,slnt,wght].ttf
- BitcountGridSingle[CRSV,ELSH,ELXP,slnt,wght].ttf
- BitcountSingleInk[CRSV,ELSH,ELXP,SZP1,SZP2,XPN1,XPN2,YPN1,YPN2,slnt,wght].ttf
- BitcountPropSingle[CRSV,ELSH,ELXP,slnt,wght].ttf
- BitcountPropSingleInk[CRSV,ELSH,ELXP,SZP1,SZP2,XPN1,XPN2,YPN1,YPN2,slnt,wght].ttf

**Comment by simoncozens** [comment]
This is confusing...

```
$ shape fonts/ttf/variable/Bitcount\[CRSV,ELSH,ELXP,slnt,wght\].ttf -u 0146
[ncommaaccent=0+600]
$ shape fonts/ttf/variable/BitcountSingle\[CRSV,ELSH,ELXP,slnt,wght\].ttf -u 0146
[n=0+600|cedillacomb=0+0]
```

The ...single... fonts do not encode ncommaaccent in the cmap. Now I am wondering what other differences there are...

**Comment by vv-monsalve** [comment]
Solved in the fonts at commit `0a89b63`
